rclpy/test/test_timer.py

```diff
@@ -133,7 +133,8 @@ def func_launch(function, args, message):
 def test_timer_zero_callbacks10hertz():
     func_launch(func_zero_callback, ['0.1'], 'received callbacks when not expected')
 
-
+# TODO(mikaelarguedas) reenable these once timer have consistent behaviour
+# on every platform at high frequency
```

Comment:
addressed in https://github.com/ros2/rclpy/pull/54/commits/426f23fc986520a0541177e7114bc07d9431ce43